src/routes/+page.server.ts

```diff
@@ -0,0 +1,34 @@
+import { fail, redirect } from '@sveltejs/kit';
+import { lucia } from '$lib/server/auth';
+
+import type { Actions, PageServerLoad } from './$types';
+
+// delete later
+export const load: PageServerLoad = async (event) => {
+	let user = null;
+	if (event.locals.user) {
+		user = event.locals.user;
+		return {
+			user,
+			loggedIn: true
+		};
+	}
+};
```

Comment:
Instead of returning loggedIn we can just return the user if it exists and change our component  so it only renders if user exists
```suggestion
export const load: PageServerLoad = async (event) => {
	if (event.locals.user) {
		return {
			user: event.locals.user,
		};
	}
};
```